<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.0" language="ru_RU">
<defaultcodec>UTF-8</defaultcodec>
<context>
    <name>QObject</name>
    <message>
        <location filename="../../src/common/net/packets/notices.cpp" line="147"/>
        <source>OK</source>
        <translation>ОК</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/notices.cpp" line="150"/>
        <source>Bad Request</source>
        <translation>Плохой запрос</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/notices.cpp" line="153"/>
        <source>Unauthorized</source>
        <translation>Неавторизован</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/notices.cpp" line="156"/>
        <source>Forbidden</source>
        <translation>Запрещено</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/notices.cpp" line="159"/>
        <source>Not Found</source>
        <translation>Не найдено</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/notices.cpp" line="162"/>
        <source>User Already Exists</source>
        <translation>Пользователь уже существует</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/notices.cpp" line="165"/>
        <source>User Not Exists</source>
        <translation>Пользователь не существует</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/notices.cpp" line="168"/>
        <source>Nick Already In Use</source>
        <translation>Ник уже используется</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/notices.cpp" line="171"/>
        <source>User Offline</source>
        <translation>Пользователь не в сети</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/notices.cpp" line="174"/>
        <source>Conflict</source>
        <translation>Конфликт</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/notices.cpp" line="177"/>
        <source>Internal Error</source>
        <translation>Внутренняя ошибка</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/notices.cpp" line="180"/>
        <source>Not Implemented</source>
        <translation>Не реализовано</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/notices.cpp" line="183"/>
        <source>Bad Gateway</source>
        <translation>Плохой шлюз</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/notices.cpp" line="186"/>
        <source>Service Unavailable</source>
        <translation>Сервис недоступен</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/notices.cpp" line="189"/>
        <source>Gateway Timeout</source>
        <translation>Шлюз не отвечает</translation>
    </message>
    <message>
        <location filename="../../src/common/net/packets/notices.cpp" line="192"/>
        <source>Unknown</source>
        <translation>Неизвестная ошибка</translation>
    </message>
</context>
<context>
    <name>SimpleSocket</name>
    <message>
        <location filename="../../src/common/net/SimpleSocket.cpp" line="207"/>
        <source>Time out</source>
        <translation>Время вышло</translation>
    </message>
</context>
<context>
    <name>Translation</name>
    <message>
        <location filename="../../src/common/Translation.cpp" line="34"/>
        <location filename="../../src/common/Translation.cpp" line="122"/>
        <source>English</source>
        <translation>Русский</translation>
    </message>
</context>
</TS>
